deployment/helm/templates/deployment.yaml

```diff
@@ -66,6 +66,11 @@ spec:
             value: "{{ .Values.trusty.endpoint }}"
           - name: "SIGSTORE_NO_CACHE"
             value: "true"
+          - name: "MINDER_DB_PASS"
+            valueFrom:
+              secretKeyRef:
+                name: minder-identity-secrets
+                key: password
```

Comment:
Let's not pass this variable this way. Instead, let's set it in the helm chart invocation itself via `extraEnv` that's defined below.